pfcpiface/main.go

```diff
@@ -28,9 +28,15 @@ var (
 
 // Conf : Json conf struct
 type Conf struct {
-	Mode        string    `json:"mode"`
-	MaxSessions uint32    `json:"max_sessions"`
-	N3Iface     IfaceType `json:"s1u"`
+	Mode        string      `json:"mode"`
+	MaxSessions uint32      `json:"max_sessions"`
+	N3Iface     IfaceType   `json:"s1u"`
+	CPIface     CPIfaceInfo `json:"cpiface"`
+}
+
+// CPIfaceInfo : CPIface interface settings
+type CPIfaceInfo struct {
+	SourceIP string `json:"zmqd_ip"`
 }
```

Comment:
remove any zmq reference. start naming `pfcp`. Figure out the IP just like s1u if not provided.